### PR TITLE
chore: add meeting minutes for monthly meeting

### PR DESCRIPTION
**Pull request by lirantal**

### CODE REVIEW

File: meetings/2021-07-21.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## Links
+
+* **Recording**:  
+* **GitHub Issue**: https://github.com/nodejs/security-wg/issues/757
+* **Minutes Google Doc**: $MINUTES_DOC$
+
+## Present
+
+* Security wg team: @nodejs/security-wg
+* mhdawson
```
Comment (mhdawson) [inline meetings/2021-07-21.md:12]:
```suggestion
Michael Dawson (@mhdawson)
```

File: meetings/2021-07-21.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Node.js  Security WorkGroup Meeting 2021-07-22
+
+## Links
+
+* **Recording**:
```
Comment (mhdawson) [inline meetings/2021-07-21.md:5]:
```suggestion
* **Recording**:  https://youtu.be/xrkn3k6p5yc
```

**Review by mhdawson — APPROVED** [review]
LGTM with a few suggestions

**Comment by lirantal** [comment]
Looks good, thanks Michael!

**Pull request merged**